components/ThreeScene.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC, Suspense, useRef } from 'react';
+import { STLLoader } from 'three/examples/jsm/loaders/STLLoader';
```

Comment:
Is there a better import? Coming from examples seems a bit weird. Not a blocker tho, so I'm approving.

Reply:
Unfortunately no there is not. Threejs package is being shipped with the example folder to provide addons, loaders and few other things to enhance the DX.

However not the best DX ever, imo.